app/digests/tests/test_digest_api.py

```diff
@@ -134,3 +135,43 @@ def test_can_filter_on_digest_stage(stage: str,
     assert response.data['total'] == 1
     assert response.data['items'][0]['stage'] == stage
 
+@pytest.mark.django_db
+def test_is_paginated(client: Client, multiple_published_digests):
+    per_page = 3
+    def _count_items(page):
+        response = client.get(f'{DIGESTS_URL}?per-page={per_page}&page={page}')
+        assert response.status_code == 200
+        assert response.data['total'] == len(multiple_published_digests)
+        items = len(response.data['items'])
+        assert items <= per_page
+        return items
+
+    last_page = 4
+    total = sum([_count_items(page) for page in range(1, last_page)]) + _count_items(page=last_page)
+
+    assert total == len(multiple_published_digests)
+
+@pytest.mark.django_db
+def test_invalid_page(client: Client):
+    response = client.get(f'{DIGESTS_URL}?page=0')
+    assert response.status_code == 404
```

Comment:
I'd expect a `400`.

Reply:
https://github.com/elifesciences/api-raml/blob/develop/src/api.raml#L150 specifies a 404 but unclear for which situations, which are 400 and which are 404?

Reply:
That's not an exhaustive list. Not match the spec (https://github.com/elifesciences/api-raml/blob/3966aeb1666f8766e104ae17561f6e6f966e2a54/src/api.raml#L122-L129) means a `400`.